api/search.go

```diff
@@ -357,7 +484,7 @@ func LegacySearchHandlerFunc(validator QueryParamValidator, queryBuilder QueryBu
 
 		nlpCriteria := getNLPCriteria(ctx, params, nlpConfig, queryBuilder, clList)
 
-		q, searchReq, countReq := CreateRequests(w, req, validator, nlpCriteria)
+		q, searchReq, countReq := CreateRequests(w, req, nlpConfig, validator, nlpCriteria)
```

Comment:
```suggestion
		q, searchReq, countReq := CreateRequests(w, req, cfg, validator, nlpCriteria)
```
should the param to this func be renamed to be consistent?